sdk/examples/wallet/create_alias.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example create_alias --release
+// In this example we will create an alias output
+// Rename `.env.example` to `.env` first
+
+use std::env;
+
+use dotenvy::dotenv;
+use iota_sdk::wallet::{Result, Wallet};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // This example uses dotenv, which is not safe for use in production
```

Comment:
Needs to be rebased and updated.

Reply:
@jlvandenhout in cased you missed it

Reply:
Ah no, Alex meant that we changed this comment in all other examples so it would be nice to adapt it

Reply:
Ah got it. Changed it.